text/src/autogluon/text/text_prediction/models/basic_v1.py

```diff
@@ -598,7 +596,7 @@ def train(self, train_data, tuning_data, resource,
             search_options=search_options,
             nthreads_per_trial=resource['num_cpus'],
             ngpus_per_trial=resource['num_gpus'],
-            checkpoint=None,
+            checkpoint=os.path.join(self._output_directory, 'checkpoint.ag'),
```

Comment:
could this possibly cause issue if 2 models are trained simultaneously in parallel during HPO?

Reply:
No, the results will be displayed correct (I've tried that).

Reply:
Wouldn't they be overwriting each-others checkpoint files?

Reply:
bumping this in case it wasn't seen due to being resolved.